bentoml/_internal/server/metrics/prometheus.py

```diff
@@ -142,3 +142,24 @@ def Gauge(self):
             self.prometheus_client.Gauge,
             registry=self.registry,
         )
+
+    @property
+    def Info(self):
+        return partial(
+            self.prometheus_client.Info,
+            registry=self.registry,
+        )
+
+    @property
+    def Enum(self):
+        return partial(
+            self.prometheus_client.Enum,
+            registry=self.registry,
+        )
+
+    @property
+    def Metric(self):
+        return partial(
+            self.prometheus_client.Metric,
+            registry=self.registry,
+        )
```

Comment:
note: adding additional metrics to expose to users.

Reply:
How should I note the change? or should I go about this another way? (i.e not making changes to `bentoml/_internal/server/metrics/prometheus.py`)